pkgs/development/libraries/glib/default.nix

```diff
@@ -70,6 +70,11 @@ stdenv.mkDerivation rec {
         url = https://gitlab.gnome.org/GNOME/glib/commit/85c4031696add9797e2334ced20678edcd96c869.patch;
         sha256 = "1hmyvhx89wip2a26gk1rvd87k0pjfia51s0ysybjyzf5f1pzw877";
       })
+      # https://gitlab.gnome.org/GNOME/glib/issues/1645
+      (fetchpatch {
+        url = https://gitlab.gnome.org/GNOME/glib/commit/e695ca9f310c393d8f39694f77471dbcb06daa9e.diff;
```

Comment:
Why not?

```suggestion
        url = https://gitlab.gnome.org/GNOME/glib/commit/e695ca9f310c393d8f39694f77471dbcb06daa9e.patch;
```

Reply:
Ah, I just hit the `plain diff` button. Not sure if it's worth the ofborg cycles though tbh :laughing: